release-notes/con-relnotes-known-issues.adoc

```diff
@@ -3,6 +3,14 @@
 
 This section lists known issues with {product} {product-version}:
 
+Catalog-import function can leak GitLab access tokens on the frontend when base64-encoded GitLab token includes a newline at the end of the string link:https://issues.redhat.com/browse/RHIDP-1734[[RHIDP-1734]]::
++
+--
+To mitigate this vulnerability until you can update to RHDH 1.1, ensure that the base64-encoded GitLab token does not contain a newline character at the end. 
```

Comment:
```suggestion
To mitigate this vulnerability until you can update to {product} 1.1, ensure that the base64-encoded GitLab token does not contain a newline character at the end of the string. 
```

Reply:
can also use {product-very-short} attribute for the approved RHDH acronym (as long as this is rendering from the attributes doc in the rhdh docs repo).